core/src/main/java/com/linecorp/armeria/client/logging/AbstractLoggingClient.java

```diff
@@ -93,6 +99,7 @@
                           Function<? super Throwable, ?> responseCauseSanitizer,
                           Sampler<? super ClientRequestContext> sampler) {
         super(requireNonNull(delegate, "delegate"));
+        this.logger = firstNonNull(logger, defaultLogger);
```

Comment:
After a merge, the default logger (`AbstractLoggingClient`) doesn't sound nice to me. How about this:

```java
if (logger != null) {
    this.logger = logger;
} else {
    this.logger = LoggerFactory.getLogger(getClass());
}
```

Reply:
fixed.
Use `LoggerFactory.getLogger(getClass());` instead of static final logger.

Reply:
fixed